sdk/voiceservices/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+  ]
+}
```

Comment:
As far as I know, only data plane packages need this file. Could we remove this?

Reply:
Sure. I added it because the initial setup required it (without it I got an error running the script that sets up the KV etc. mentioned in the docs), so I assumed that all SDK packages needed it. But it looks like the tests now pass without it again, so I'll remove it.